src/rcheevos.filters

```diff
@@ -0,0 +1,123 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
i don't believe this file is used. please delete it.

Reply:
Must have been leftover. It's gone.